src/lsst/cst/conversions/tools.py

```diff
@@ -51,214 +46,87 @@ def data_id_to_str(data_id: dict):
     Returns
     -------
     data_id_str: `str`
-        Data identifier string
-    """
-    cal_exp_id = CalExpId(**data_id)
-    return str(cal_exp_id)
-
-
-def sort_dataframe(
-    df: pd.DataFrame,
-    sort_key: str,
-    ascending: bool = False,
-    set_index: bool = True,
-) -> pd.DataFrame:
-    """Return a sorted copy of the dataframe 'df' by index,
-    selecting the desired order (ascending or descending)
-    using the ascending argument, it also exchanges the index
-    of the dataframe for the sort_key if set_index parameter
-    is True (default).
-
-    Parameters
-    ----------
-    df : `pandas.DataFrame`
-        dataframe to be sorted
-    sort_key: `str`
-        column key used to sort the dataframe
-    ascending: `bool`, optional
-        ascending/descending sorting
-    set_index: `bool`, optional
-        set sorted key as the dataframe index
-
-    Returns
-    -------
-    result : `pandas.DataFrame`
-        Copy of the `~pandas.DataFrame` sorted using the selected column.
+        Data identifier string.
     """
-    if sort_key not in df.columns:
-        raise Exception(f"Index {sort_key} not existing in the dataframe")
-    df = df.sort_values(sort_key, ascending=ascending)
-    if set_index:
-        df.set_index(np.array(range(len(df))), inplace=True)
-    return df
+    data_id_str = json.dumps(data_id)
+    return data_id_str
 
 
-def shuffle_dataframe(df: pd.DataFrame, random_state: int = 0) -> pd.DataFrame:
-    """Return a copy of the dataframe df shuffled, random_state
-    argument may be used to reproduce same shuffling.
+def psf_size_at_pixel_xy(psf, bbox, xy):
```

Comment:
add type of the arguments and if able the return type also:

def psf_size_at_pixel_xy(psf: x, bbox: y, xy: z) -> X:

Reply:
Hmm, I can do for `xy: (int, int)`, but the linter will not take `psf: lsst.meas.extensions.psfex.PsfexPsf` or `bbox: lsst.geom.Box2I`. Thoughts? 

And I added `-> dict` here, and to the others (or `-> str`).

Reply:
Tim's response resolved this for me.